importer/api/__init__.py

```diff
@@ -0,0 +1,100 @@
+"""
+Import OLX data into LORE.
+"""
+
+# stdlib
+from glob import glob
+from shutil import rmtree
+from tempfile import mkdtemp
+from os.path import join
+import tarfile
+import zipfile
+
+# PyPi
+
+from xbundle import XBundle, DESCRIPTOR_TAGS
+from lxml import etree
+
+# MITODL
+# commented out for now, until it's merged
+# from lore.learningobjects.models import Course, LearningObject
+
+def import_course_from_file(filename):
+    """
+    Import OLX from .zip or tar.gz.
+    """
+    tempdir = mkdtemp()
+    if filename.endswith(".tar.gz"):
+        course = tarfile.open(filename, "r")
+    elif filename.endswith(".zip"):
+        course = zipfile.ZipFile(filename, "r")
+    else:
+        raise ValueError("Unexpected file type (want tarball or zip).")
+    course.extractall(tempdir)
+    dirs = glob(join(tempdir, "*"))
+    assert len(dirs) == 1
```

Comment:
no asserts in prod code (ok in tests), as they don't always run depending on how the python interpreter is invoked